service/src/main/kotlin/xyz/lebkuchenfm/domain/eventstream/Event.kt

```diff
@@ -12,6 +12,11 @@ sealed interface Event {
         val songs: List<Song>,
     ) : Event
 
+    data class Skip(
```

Comment:
We can store `SkipAmount` instance here, and map it to DTO later. We don't need `all` and `amount` as separate fields in domain. I would also suggest that we change the DTO to match this - later.